EDF.py

```diff
@@ -8,32 +41,44 @@
 import re
 import warnings
 
+
 def padtrim(buf, num):
     num -= len(buf)
-    if num>=0:
-        # pad the input to the specified length
-        return str(buf) + ' ' * num
-    else:
-        # trim the input to the specified length
-        return buf[0:num]
-
-####################################################################################################
-# the EDF header is represented as a tuple of (meas_info, chan_info)
-# meas_info should have ['record_length', 'magic', 'hour', 'subject_id', 'recording_id', 'n_records', 'month', 'subtype', 'second', 'nchan', 'data_size', 'data_offset', 'lowpass', 'year', 'highpass', 'day', 'minute']
-# chan_info should have ['physical_min', 'transducers', 'physical_max', 'digital_max', 'ch_names', 'n_samps', 'units', 'digital_min']
-####################################################################################################
+    return (str(buf) + ' ' * num) if (num >= 0) else buf[0:num]
+
+
+def writebyte(file, content, encoding='utf8'):
```

Comment:
```suggestion
def writebyte(file, content, encoding='ascii'):
```

`writebyte` should default to using ASCII encoding because that's what the EDF spec [actually requires](https://www.edfplus.info/specs/edf.html).

Reply:
I remember you asking me to do this a year ago. I forgot. Apologies. :sweat_smile: